### ISSUE TITLE
Add Password Recovery

### ISSUE THREAD

**Issue by chenblair** [k15z/bayms.web#40]
maybe email the user with a new randomized password

**Comment by k15z** [comment]
Good idea. The server is already configured to send mail, but the sent messages will most likely end up sorted into the spam folder since I didn't bother setting up MX records. Although, instead of sending a new password, maybe it would be better to send a password reset link?
